Project2/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">  
+    <title>MovieCard</title>
+</head>
+<body style="background-color:#8a2be2"; >
+<div class="card"> 
+<figure>
+<a href="https://www.youtube.com/watch?v=DYYtuKyMtY8"><img class="thumbnail" src="images\SpiderManFFHTrailer-01.png" alt="Spider-Man Far From Home Trailer"></a>
```

Comment:
You want the other kind of slash here in your image path:
```suggestion
<a href="https://www.youtube.com/watch?v=DYYtuKyMtY8"><img class="thumbnail" src="images/SpiderManFFHTrailer-01.png" alt="Spider-Man Far From Home Trailer"></a>
```